examples/with-mobx-keystone-typescript/store/index.tsx

```diff
@@ -0,0 +1,58 @@
+import { FC, createContext, useState, useContext } from 'react'
+import { useStaticRendering } from 'mobx-react-lite'
+import {
+  registerRootStore,
+  isRootStore,
+  SnapshotInOf,
+  fromSnapshot,
+} from 'mobx-keystone'
+
+import { RootStore } from './root'
+
+const isServer = typeof window === 'undefined'
+// eslint-disable-next-line react-hooks/rules-of-hooks
+useStaticRendering(isServer)
+
+let store: RootStore | null = null
+
+export const initStore = (snapshot?: SnapshotInOf<RootStore>) => {
+  if (isServer) {
```

Comment:
Add `typeof window === 'undefined'` here and remove the `isServer` variable, that way webpack can remove unrequired code in the server/browser builds